diff --git a/exultmenu.cc b/exultmenu.cc
index 8873159a7..7d87917aa 100644
--- a/exultmenu.cc
+++ b/exultmenu.cc
@@ -399,7 +399,6 @@ BaseGameInfo* ExultMenu::run() {
 				= "Please add the games in File Sharing";
 		const char close_screen_msg[] = "Touch screen for help!";
 #else
-		const char game_missing_msg[] = "Please edit the configuration file.";
 		const char close_screen_msg[] = "Press ESC to exit.";
 #endif
 		// Create our message and programatically center it.
@@ -408,9 +407,9 @@ BaseGameInfo* ExultMenu::run() {
 				"",
 				"To play you need the static data for either",
 				R"("The Black Gate" or "Serpent Isle".)",
-				game_missing_msg,
-				"",
-				"",
+				"Please copy the GAMEDAT and STATIC directories",
+				"from either to the appropriate game directory in",
+				"$HOME/.var/app/info.exult.exult/data/",
 				close_screen_msg,
 		};
 		const int total_lines
